mining-buddy.lic

```diff
@@ -95,7 +96,11 @@ class MiningBuddy
   def mine_rooms(rooms)
     rooms.each do |room|
       wait_for_script_to_complete('safe-room') if bleeding?
-      check_repair if mine?(room)
+      if mine?(room)
+        check_repair
+        wait_for_script_to_complete('buff', ['mining']) if @settings.waggle_sets['mining']
+        bput('speculate luck', '^You focus your mind on the world around you', '^You are already focusing on the world around you.', '^Your pattern-matching skills are still exhausted') if DRStats.trader? && DRStats.circle >= 65
```

Comment:
💡 This might be a good candidate for `DRCA`

Reply:
I can't remember for sure if DRCA can access `DRStats`, but I'm pretty sure it can.